Makefile

```diff
@@ -44,7 +44,7 @@ load-images:  ## Load images into the local kubernetes kind cluster.
 .PHONY: helm-install
 helm-install: start-kind cert-manager load-images  ## This is a phony target that is used to install the Tobs Helm chart.
 	helm dep up chart/
-	helm install --wait --timeout 15m test chart/
+	helm install --wait --timeout 15m test chart/ --values chart/values.yaml --values chart/ci/default-values.yaml
```

Comment:
I added this as during development we should always apply `chart/ci/default-values.yaml` to remove components which are not welcome on `kind` clusters.